fw/vhost.c

```diff
@@ -1903,6 +1911,11 @@ __tfw_cfgop_frang_http_methods(TfwCfgSpec *cs, TfwCfgEntry *ce,
 		methods_mask |= (1UL << method_id);
 	}
 
+	if (!ce->val_n && !ce->dflt_value) {
```

Comment:
You don't need to check !ce->dflt_value here, because if it is dflt_value vam_n is always 3 (get post head).  This check is better to do before `TFW_CFG_ENTRY_FOR_EACH_VAL` loop

Reply:
TFW_CFG_CHECK_VAL_N(>, 0, cs, ce);
TFW_CFG_CHECK_NO_ATTRS(cs, ce);

is right way. Tempesta interpret get=hhhsdsd as attribute and silently ignore such invalid configuration

Reply:
Reworked.